Mathlib/Analysis/SpecialFunctions/Log/Basic.lean

```diff
@@ -219,6 +219,13 @@ theorem log_injOn_pos : Set.InjOn log (Set.Ioi 0) :=
   strictMonoOn_log.injOn
 #align real.log_inj_on_pos Real.log_injOn_pos
 
+theorem log_lt_sub_one_of_pos (hx1 : 0 < x) (hx2 : x ≠ 1) : log x < x - 1 := by
+  have h : log x ≠ 0
+  · rwa [← log_one, log_injOn_pos.ne_iff hx1]
+    norm_num
```

Comment:
Did the mathlib3 proof not work here? I'm puzzled by where the `norm_num` came from.

Reply:
I think Heather has just improved the proof.